LLM/config.ts

```
@@ -98,7 +98,83 @@ export function editModel(
+  { name: "Claude 3.5 Haiku", id: "claude-3-5-haiku-latest" },
+  { name: "Claude 3 Opus", id: "claude-3-opus-latest" },
+  { name: "Claude 3 Sonnet", id: "claude-3-sonnet-latest" },
+  { name: "Claude 3 Haiku", id: "claude-3-haiku-latest" },
+];
+
+const OllamaModels: LLMModel[] = [
+  { name: "Gemma 3 4B", id: "gemma3:4b" },
+  { name: "Qwen 3 VL 4B", id: "qwen3-vl:4b" },
+  { name: "Deepseek OCR 3B", id: "deepseek-ocr:3b" },
+];
+export const AvailableLLMs: Record<string, LLMModel[]> = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'AvailableLLMs'. Consider to delete it or rename it.

'AvailableLLMs' is defined here:


(lint/suspicious/noRedeclare)